spec/unit/lib/cloud_controller/errands/rotate_database_key_spec.rb

```diff
@@ -36,6 +36,7 @@ module VCAP::CloudController
             'VCAP::CloudController::PackageModel' => PackageModel.make(:docker, package_hash: Sham.guid, error: 'a-error', docker_image: 'image', docker_username: 'user'),
             'VCAP::CloudController::DropletModel' => DropletModel.make(:all_fields),
             'VCAP::CloudController::BuildpackLifecycleDataModel' => BuildpackLifecycleDataModel.make(:all_fields),
+            'VCAP::CloudController::CNBLifecycleDataModel' => CNBLifecycleDataModel.make(:all_fields),
```

Comment:
Can be removed if `CNBLifecycleDataModel` does not need any encrypted fields (see other comment). This would also fix the backward compatibility tests.

Reply:
This should be still removed as as `CNBLifecycleDataModel` doesn't have any encrypted field.